src/Resources/contao/library/Contao/Controller.php

```diff
@@ -139,15 +139,7 @@ public static function getTemplateGroup($strPrefix)
 							foreach ($arrThemeTemplates as $strFile)
 							{
 								$strTemplate = basename($strFile, strrchr($strFile, '.'));
-
-								if (!isset($arrTemplates[$strTemplate]))
-								{
-									$arrTemplates[$strTemplate][] = $objTheme->name;
-								}
-								else
-								{
-									$arrTemplates[$strTemplate][] = $objTheme->name;
-								}
+								$arrTemplates[$strTemplate][] = $objTheme->name;
```

Comment:
Was the old code originally maybe supposed to look like:
```php
if (!isset($arrTemplates[$strTemplate]))
{
	$arrTemplates[$strTemplate] = array();
}
$arrTemplates[$strTemplate][] = $objTheme->name;
```
This way no notice would be issued when the array does not exist. IMO it would be good if we would reduce the amount of notices to an absolute minimum.
Therefore, when you want to touch this file, please in a manner where no notice is issued, otherwise this chunk can be omitted as unrelated to the PR.